### ISSUE TITLE
Little layout issue at the 'Contact us'

### ISSUE THREAD

**Issue by ibagur** [vidonne/unhcrdown#16]
Hi,

I noticed this on the **Paged simple** template. After you define the metadata at the YAML header:

```
author:
  - name: John Doe
    affiliation: Poetry Officer
    email: johndoe@unhcr.org
```

The `email` field seems it gets picked twice at render time, so I'm getting the following output where it gets also added right after the `affiliation`:

---
CONTACT US

John Doe
**Poetry Officerjohndoe@unhcr.org**
johndoe@unhcr.org

www.unhcr.org
---

**Comment by vidonne** [comment]
Hi @ibagur we are aware of this. It is mentioned in issue #2
But no solution yet unfortunately

**Comment by ibagur** [comment]
Hi @vidonne , sorry! I didn't noticed it was already mentioned. Please dismiss my issue then. Apart from this, the template works great!

**Comment by vidonne** [comment]
fix with commit 58f2f43
